PayPal/src/main/java/com/braintreepayments/api/paypal/RecurringBillingAmountBreakdown.kt

```diff
@@ -0,0 +1,26 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * A recurring billing amount breakdown.
+ *
+ * @property itemTotal - Total amount of the items before any taxes or discounts
+ * @property insurance - (optional) Cost of insurance applied to the shipment or items
+ * @property discount - (optional) General discount applied to the total transaction
+ * @property handling - (optional) Cost associated with handling the items (e.g., packaging or processing)
+ * @property shippingDiscount - (optional) Discount amount applied specifically to shipping
+ * @property shippingTotal - (optional) Cost of shipping the items
+ * @property taxTotal - (optional) Total tax amount applied to the transaction
```

Comment:
Can we re-order the properties in this object to the way they exist in the design doc for clarity and to group together the properties that _cannot_ be passed with `PayPalRecurringBillingDetails`? We should call out in the doc strings which parameters cannot be passed when `PayPalRecurringBillingDetail`s are present.